extra/modules/greenbids-real-time-data/src/main/java/org/prebid/server/hooks/modules/greenbids/real/time/data/config/DatabaseReaderFactory.java

```diff
@@ -49,6 +51,41 @@ public void initialize(Promise<Void> initializePromise) {
         .onComplete(initializePromise);
     }
 
+    private Path downloadFile(String url, String accountId, String licenseKey) throws IOException {
+        final URL downloadUrl = new URL(url + "&account_id=" + accountId + "&license_key=" + licenseKey);
+        final HttpURLConnection connection = (HttpURLConnection) downloadUrl.openConnection();
+        connection.setRequestMethod("GET");
+
+        final String auth = accountId + ":" + licenseKey;
+        final String encodedAuth = Base64.getEncoder().encodeToString(auth.getBytes());
+        connection.setRequestProperty("Authorization", "Basic " + encodedAuth);
```

Comment:
I suggest using vertx http client for file downloading

Also, it's a little bit weird that it's required to send credentials via the url and the authorization header simultaneously, I assume it should work both ways but separately, so I prefer using just URL approach and do not expose account id and license key as separate properties